project/V.scala

```diff
@@ -70,18 +70,12 @@ object V {
   // Scala 2
   // whenever version is removed please add it to MtagsResolver under last supported Metals version
   def deprecatedScala2Versions = Seq(
-    scala211,
-    "2.12.11",
-    "2.12.12",
-    "2.12.13",
-    "2.12.14",
-    "2.13.5",
-    "2.13.6",
-    "2.13.7",
+    "2.12.11", "2.12.12", "2.12.13", "2.12.14", "2.13.5", "2.13.6", "2.13.7",
     "2.13.8",
   )
 
   def nonDeprecatedScala2Versions = Seq(
+    scala211,
```

Comment:
I would still keep it as deprecated, but add a separate job running on JDK 8 that tests this explicitely.